src/T2DMap.cpp

```diff
@@ -102,9 +102,11 @@ T2DMap::T2DMap(QWidget* parent)
     QStringList headerLabels;
     headerLabels << tr("Room Id") << tr("Room Name");
     mMultiSelectionListWidget.setHeaderLabels(headerLabels);
-    mMultiSelectionListWidget.setToolTip(
-            tr("<html><head/><body><p>Click on a line to select or deselect that room number (with the given name if the rooms are named) to add or remove the room from the selection.  Click on the "
-               "relevant header to sort by that method.  Note that the name column willl only show if at least one of the rooms has a name.</p></body></html>"));
+    mMultiSelectionListWidget.setToolTip(QStringLiteral("<html><head/><body>%p</body></html>")
+                                         .arg(tr("<p>Click on a line to select or deselect that room number (with the given name if the "
+                                                 "rooms are named) to add or remove the room from the selection.  Click on the "
+                                                 "relevant header to sort by that method.  Note that the name column willl only "
+                                                 "show if at least one of the rooms has a name.</p>")));
```

Comment:
I deliberately make an exception for paragraph tags because if there is more than one it is better to not hide just the first opening and last closing ones - on the basis that it causes *less* surprise then if there is a `</p><p>` in the middle of a string.

Reply:
Fair enough, but there are no such in the middle here? 
I want to outsource these html marks as many and as often as possible. 
Agreed on avoiding the surprising paragraph tags. In those cases where they are in the middle, they should stay on the edges as well. But in this case, no tags are needed for translation at all, at all.